### DISCUSSION TITLE
直播检测到用户离开，自动暂停直播

### DISCUSSION THREAD

**Discussion by suibianguangguang77** [the1812/Bilibili-Evolved#4932] [提问 / Q&A]
看直播时长时间不操作，直播间会提醒：检测到您已离开当前屏幕，倒计时后即将暂停播放。有没有插件可以规避这项暂停机制

**Comment by the1812** [comment]
https://github.com/the1812/Bilibili-Evolved/issues/1159#issuecomment-721810895
